README.md

```diff
@@ -1,10 +1,20 @@
 # openSUSE with Java and Tomcat image
 
-This project builds on the openSUSE Java 8 image [here](https://github.com/CAFapi/opensuse-java8-images) to build a pre-configured Tomcat Docker image.
+This project consists of 2 separate images which build on the openSUSE Java 8 image [here](https://github.com/CAFapi/opensuse-java8-images) 
+to build two pre-configured Tomcat.
 
-It can be used as a base image for hosting web projects which use Java technologies such as Java Servlets or JavaServer Pages.
+One that uses CAF Logging framework and the other that relies on Tomcat Jul logging.
 
-Here is an example Dockerfile which uses this image as a base:
+Each one can be used as a base image for hosting web projects which use Java technologies such as Java Servlets or JavaServer Pages.
+
+Here is an example Dockerfile which uses the image with Tomcat Jul logging as a base:
+
+    FROM cafapi/opensuse-tomcat-jul:latest
```

Comment:
change to
Juli logging as a base:

    FROM cafapi/opensuse-tomcat-juli:latest